Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -271,15 +312,39 @@ private void ContentGrid_ManipulationDelta(object sender, ManipulationDeltaRoute
                     _leftCommandTransform.TranslateX = 20;
```

Comment:
This value is being used when u swipe more than activationwidth and sticks the content. I didn't understand what you're saying though. It's looking cool though. Current original source still has this value. Should I remove it or not ? We can discuss.